docs/design/#3-View-Personalized-Recommendations-of-High-Schools.txt

```diff
@@ -0,0 +1,33 @@
+title #3-View-Personalized-Recommendations-of-High-Schools
+
+actor User
+participant UI (Browser)
+participant API
+participant Data Store
+
+activate UI (Browser)
+User->UI (Browser): Set Preferences
+activate API
+UI (Browser) -> API: Send Preferences
+activate Data Store
+API -> Data Store: Store Preferences
+API -> API: Choose Random School IDs (based on location?)
+API -> Data Store: Request Simple data for schools
+Data Store -> API: Send Schools' Simplified Data 
+API --> UI (Browser): Send Selections (Simple View Dataset)
+UI (Browser) --> User: Render Simple View
+Data Store -> API: Confirm Stored Preferences
```

Comment:
Is this just a "status ok" message to the API? Should it go directly under "Store Preferences" (line 13)?

Reply:
It's confirming there's no pending change to user preference also initiate API to start calculating user preference